test/functional/interface_zmq.py

```diff
@@ -484,11 +481,12 @@ def test_mempool_sync(self):
         # Things continue to happen in the "interim" while waiting for snapshot results
         # We have node 0 do all these to avoid p2p races with RBF announcements
         for _ in range(num_txs):
-            txids.append(self.nodes[0].sendtoaddress(address=self.nodes[0].getnewaddress(), amount=0.1, replaceable=True))
-        self.nodes[0].bumpfee(txids[-1])
+            txs.append(self.wallet.send_self_transfer(from_node=self.nodes[0]))
+        txs[-1]['tx'].vout[0].nValue -= 1000
+        self.nodes[0].sendrawtransaction(txs[-1]['tx'].serialize().hex())
```

Comment:
I recall that you asked on IRC whether it makes sense to introduce a bumpfee method for the wallet. On a second thought, I think it could make sense.

Currently it looks like `send_self_transfer` will put an utxo in the wallet which will never exist if the tx is bumped?

So a helper could drop the utxo(s?) and even add the new ones?

Reply:
yep, I think it would be a useful method to have. I'll open a follow-up PR for adding it as a method to `MiniWallet`